iris-mpc/src/bin/server.rs

```diff
@@ -777,27 +779,47 @@ async fn server_main(config: Config) -> eyre::Result<()> {
     let is_ready_flag = Arc::new(AtomicBool::new(false));
     let is_ready_flag_cloned = Arc::clone(&is_ready_flag);
 
-    #[derive(Serialize, Deserialize)]
+    #[derive(Debug, Serialize, Deserialize, Clone)]
     struct ReadyProbeResponse {
         image_name: String,
         uuid:       String,
+        shutdown:   bool,
```

Comment:
a nit suggestion on naming:
```suggestion
        shutting_down:   bool,
```